java/ql/src/semmle/code/java/dataflow/internal/DataFlowImplCommon.qll

```diff
@@ -54,14 +77,13 @@ private module ImplCommon {
   }
 
   /**
-   * Holds if `arg` flows through `call` using only value-preserving steps.
+   * Holds if `arg` flows through a call to `out` using only value-preserving steps.
```

Comment:
As-is this is ambiguous (and I read it wrong at first, reading "a call to `out`"), so perhaps this should be "flows to `out` through a call" instead.